activejob/lib/active_job/arguments.rb

```diff
@@ -72,8 +72,14 @@ def deserialize(arguments)
 
       def serialize_argument(argument)
         case argument
-        when *PERMITTED_TYPES
```

Comment:
`PERMITTED_TYPES` is now only used in `deserialize_argument`, perhaps we can get rid of it.

Reply:
Thanks, deleted.